### PR TITLE
docs: refactor SSO docs

### PR DESCRIPTION
**Pull request by sarahschwartz**

### CODE REVIEW

File: content/00.zksync-era/30.unique-features/30.zksync-sso/28.account-recovery.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Multiple Devices with Passkey Synchronization
+
+Users can synchronize their passkeys across multiple devices using services like iCloud Keychain for Apple devices or similar services on other platforms.
+This means if one device is lost, the passkeys are still accessible on other synchronized devices.
+
+## Other Options
+
+*More account recovery options are coming soon.*
+
+<!--
```
Comment (popzxc) [inline content/00.zksync-era/30.unique-features/30.zksync-sso/28.account-recovery.md:17]: Are commented out blocks intentional?

**Review by popzxc — COMMENTED** [review]
Quickly glanced through the changes -- looks indeed more readable to a person who's not very familiar with SSO.
Left one question.

File: content/00.zksync-era/30.unique-features/30.zksync-sso/23.sessions.md [hunk 2]
```diff
@@ -94,9 +82,34 @@ type Limit =
...
+### `LimitType` Interface
+
+There are three types of limits:
 
-Allowed ETH transfers (non-contract calls).
+- **Unlimited**: No limit is enforced.
+- **Lifetime**: Limit is enforced over the lifetime of the session.
+- **Allowance**: Limit is enforced over a specific period of time and is reset every `period` seconds (based on the `block.timestamp`).
+
+::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
```
Comment (JackHamer09) [inline content/00.zksync-era/30.unique-features/30.zksync-sso/23.sessions.md]: Btw we can already remove this

File: content/00.zksync-era/30.unique-features/30.zksync-sso/28.account-recovery.md (hunk 1 above)
Comment (sarahschwartz) [reply content/00.zksync-era/30.unique-features/30.zksync-sso/28.account-recovery.md:17]: They were left there from before

**Review by popzxc — APPROVED** [review]

**Review by JackHamer09 — APPROVED** [review]

**Pull request merged**